backend/swagger/v1/swagger.yaml

```diff
@@ -355,7 +355,22 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/CreateAnswerRequestBody'
-
+  '/api/v1/seed_database':
+      get:
+        summary: Get seeds
+        description: "Get ready seeds"
```

Comment:
I know other descriptions are not good, but let's start improving them soon.  Please improve this one.

Reply:
@PolinaStamenova